images/base/overlay/etc/cont-init.d/15-setup_devices.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+gow_log "**** Configure devices ****"
+
+# make sure we're in the right groups to use all the required devices
+# we're actually relying on word splitting for this call, so disable the
+# warning from shellcheck
+# shellcheck disable=SC2086
```

Comment:
i think this shellcheck comment needs to be directly above the line it applies to (line 12)

Reply:
Oops, that is correct. I must have put the log line in afterwards and messed that up